Add interfaces/webrtc-priority.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/51c8741/ed/idl/webrtc-priority.idl
Build: https://travis-ci.org/w3c/webref/builds/729295603

foolip commented:
I've sent https://github.com/web-platform-tests/wpt/pull/21462 with a test added.

foolip commented:
Nobody has implemented the API for this yet. https://github.com/web-platform-tests/wpt/pull/21462 added tests, and that PR can be revived if somebody does implement. Until then, this PR should be left open to prevent the bot from creating a new one.

autofoolip committed:
Add interfaces/webrtc-priority.idl

Source: https://github.com/w3c/webref/blob/51c8741/ed/idl/webrtc-priority.idl
Build: https://travis-ci.org/w3c/webref/builds/729295603
